app/src/main/java/com/gh4a/fragment/RepositoryListContainerFragment.java

```diff
@@ -16,19 +16,45 @@
 import android.view.SubMenu;
 import android.view.View;
 import android.view.ViewGroup;
-
 import com.gh4a.BaseActivity;
 import com.gh4a.Gh4Application;
 import com.gh4a.R;
 import com.gh4a.loader.LoaderCallbacks;
 import com.gh4a.utils.ApiHelpers;
 import com.gh4a.widget.SwipeRefreshLayout;
-
 import org.eclipse.egit.github.core.Repository;
 
 public class RepositoryListContainerFragment extends Fragment implements
         LoaderCallbacks.ParentCallback, SearchView.OnCloseListener, SearchView.OnQueryTextListener,
         MenuItemCompat.OnActionExpandListener, SwipeRefreshLayout.ChildScrollDelegate {
+    private static final String EXTRA_USER = "user";
+    private static final String EXTRA_IS_ORG = "is_org";
+    private static final String EXTRA_FILTER_TYPE = "filter_type";
+    private static final String STATE_KEY_FILTER_TYPE = "filter_type";
+    private static final String STATE_KEY_SORT_ORDER = "sort_order";
+    private static final String STATE_KEY_SORT_DIRECTION = "sort_direction";
+    private static final String STATE_KEY_SEARCH_VISIBLE = "search_visible";
+    private static final String STATE_KEY_QUERY = "search_query";
+    private static final String STATE_KEY_SEARCH_IS_EXPANDED = "search_is_expanded";
+    private static final String FILTER_ALL = "all";
+    private static final String OWNER = "owner";
```

Comment:
These will probably need some prefix or be moved to another file.
@maniac103

Reply:
**FilterConstants** file is good ?

Reply:
I have found that [RepositoryService](https://github.com/slapperwan/gh4a/blob/master/github-api/src/main/java/org/eclipse/egit/github/core/service/RepositoryService.java) already contains some of these so the rest should be moved here too.

Reply:
For the naming. Filter constants should start with TYPE_ as these already present in RepositoryService. Sort constants should be prefixed with SORT_. And the main, search tag are specific to this fragment so these 2 leave here.